.vscode/launch.json

```diff
@@ -4,156 +4,216 @@
         {
             "type": "node",
             "request": "launch",
-            "name": "iOS - Preview",
```

Comment:
I renamed and sorted the command in this file such that the commands start with the command name instead of the platform.